anda/system/cros-keyboard-map/cros-keyboard-map.spec

```diff
@@ -53,15 +53,15 @@ mkdir -p %buildroot/etc/systemd/system
 tee %buildroot/etc/systemd/system/cros-keyboard-map.service <<EOF
 [Unit]
 Description=Generate chromebook keyboard layout
-Before=keyd.service
 After=tmp.mount
 
 [Service]
 Type=oneshot
 ExecStart=/bin/bash /usr/bin/um-generate-cros-keymap
+ExecStartPost=+/usr/bin/systemctl restart keyd.service
```

Comment:
`keyd reload`?

Reply:
restart should do the same things in practice